bundles/org.openhab.binding.wemo/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -1,429 +1,429 @@
+			<channel id="timeRemaining" typeId="timeRemaining" />
+			<channel id="waterLevelReached" typeId="waterLevelReached" />
+			<channel id="cleanAdvise" typeId="cleanAdvise" />
+			<channel id="filterAdvise" typeId="filterAdvise" />
+			<channel id="brewed" typeId="brewed" />
+			<channel id="lastCleaned" typeId="lastCleaned" />
+		</channels>
+
+		<config-description>
+			<parameter name="udn" type="text">
+				<label>Unique Device Name</label>
+				<description>The UDN identifies the WeMo Device</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="false" min="15" max="180">
+				<label>Polling Interval</label>
+				<description>Interval polling the WeMo Coffee Maker.
+				</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="dimmer">
+		<label>WeMo DimmerSwitch</label>
+		<description>This is a WeMo DimmerSwitch</description>
+
+		<channels>
+			<channel id="brightness" typeId="brightness" />
+			<channel id="faderCountDownTime" typeId="faderCountDownTime" />
+			<channel id="faderEnabled" typeId="faderEnabled" />
+			<channel id="timerStart" typeId="timerStart" />
+			<channel id="nightMode" typeId="nightMode" />
+			<channel id="startTime" typeId="startTime" />
+			<channel id="endTime" typeId="endTime" />
+			<channel id="nightModeBrightness" typeId="nightModeBrightness" />
+		</channels>
+
+		<config-description>
+			<parameter name="udn" type="text">
+				<label>Unique Device Name</label>
+				<description>The UDN identifies the WeMo Device</description>
+				<required>true</required>
+			</parameter>
+
+		</config-description>
+	</thing-type>
+
+	<channel-type id="state">
+		<item-type>Switch</item-type>
+		<label>Switch</label>
+		<description>Turns the power on or off</description>
+		<category>Switch</category>
+	</channel-type>
+
+	<channel-type id="motionDetection">
+		<item-type>Switch</item-type>
+		<label>Motion Status</label>
+		<description>Indicates whether motion is detected or not</description>
+		<category>Switch</category>
+		<state readOnly="true"></state>
+	</channel-type>
+
+	<channel-type id="lastMotionDetected" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Last Activity (Date/Time)</label>
+		<description>Date/time when last motion was detected</description>
+	</channel-type>
+
+	<channel-type id="lastChangedAt" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Last Activity (Date/Time)</label>
+		<description>Date/time when the state last changed</description>
+	</channel-type>
+
+	<channel-type id="lastOnFor" advanced="true">
+		<item-type>Number</item-type>
+		<label>Last Activity (s)</label>
+		<description>Duration the device has been switched on for</description>
+		<state pattern="%.1f s" />
+	</channel-type>
+
+	<channel-type id="onToday" advanced="true">
+		<item-type>Number</item-type>
+		<label>Today's Activity (s)</label>
```

Comment:
the unit is already in the pattern so it might be better to put in the description, probably written out as `in seconds`. (A alternative would be to model these channels as UoM so the user can specify the what unit to display it in)

Reply:
It is already in the label  (s)